engine/datamanager.go

```diff
@@ -317,6 +317,25 @@ func (dm *DataManager) CacheDataFromDB(prfx string, ids []string, mustBeCached b
 	return
 }
 
+func (dm *DataManager) GetAccount(id string) (acc *Account, err error) {
+	acc, err = dm.dataDB.GetAccountDrv(id)
+	if err != nil {
```

Comment:
we should only query remote if the error is not found, not for all errors. This is valid for all remote methods